app-misc/jigdo/jigdo-0.7.3.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="Ease the distribution of files"
+DESCRIPTION="Jigsaw Download, or short jigdo, is a tool designed to \
+ease the distribution of very large files over the internet, for example \
+CD or DVD images. Its aim is to make downloading the images as easy for users \
+as a click on a direct download link in a browser, while avoiding all the \
+problems that server administrators have with hosting such large files."
+HOMEPAGE="http://atterer.org/jigdo/"
+COPYRIGHT="2001-2006 Richard Atterer"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://atterer.org/sites/atterer/files/2009-08/jigdo/jigdo-$portVersion.tar.bz2"
+CHECKSUM_SHA256="875c069abad67ce67d032a9479228acdb37c8162236c0e768369505f264827f0"
+PATCHES="jigdo-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 ?x86 x86_64"
```

Comment:
I think you can strike x86_gcc2 from the list, x86_64 builds ok

Reply:
Maybe add SECONDARY_ARCHITECTURES="x86" and add $secondaryArchSuffix to the needed places?